delta/messages.adoc

```diff
@@ -26,13 +26,16 @@ Same format as <<{m3}.adoc#identifiers, identifiers>>; must be unique per <<part
 [[queryIdType]]QueryId:: Id of a query.
 Same format as <<{m3}.adoc#identifiers, identifiers>>; must be unique per <<participation>>.
 
-[[chunkType]]SerializationChunk:: <<{serialization}.adoc, Chunk of nodes>>.
+[[chunkType]]SerializationChunk:: Chunk of nodes as described in <<{serialization}.adoc, serialization specification>>.
 
 [[eventType]]Event:: Any <<events, event>>.
 
 [[participationIdType]]ParticipationId:: Id of a <<participation>>.
 Same format as <<{m3}.adoc#identifiers, identifiers>>; must be unique for all concurrent participations.
 
+[[clientIdType]]clientId:: Id of a <<client>>.
```

Comment:
With this, it'd be more logical if the scenarios – e.g. the first one – would use "`participation-a`" for the `participationId` rather than "`client-a`".

Reply:
done in #386